dynamodb/src/main/scala/zio/dynamodb/ConditionExpression.scala

```diff
@@ -69,12 +108,21 @@ object ConditionExpression {
       GreaterThanOrEqual(self, Operand.ValueOperand(t.toAttributeValue(that)))
```

Comment:
```suggestion
      GreaterThan(self, Operand.ValueOperand(t.toAttributeValue(that)))
```
oooops - this is a bug of mine - cut and paste error